sys/vane/jael.hoon

```diff
@@ -2137,11 +2133,11 @@
     ::  keep the following in sync with ++extract-snap:file:su
     ::
     %=    +>.$
-        eve.urb       eve.snap
-        etn           etn.snap(source source.etn)
-        kyz.puk.sub   kyz.snap
-        +.eth.sub     eth.snap
-        sap           sap(last-block 0)
+        heard.etn         heard.snap
+        latest-block.etn  latest-block.snap
+        kyz.puk.sub       kyz.snap
+        +.eth.sub         eth.snap
```

Comment:
Since we manually specify `[dns hul]:eth.sub` in `++extract-snap`, arguably we should refer to them specifically here as well.

Reply:
Good call. I'll fix that.